pistachio: Use hard-float by default

francois-berder-imgtec opened the pull request:
This connects #164 .

Signed-off-by: Francois Berder <francois.berder@imgtec.com>

Ham22 commented on target/linux/pistachio/creator-platform-default-cascoda.config:
```diff
@@ -11,3 +11,6 @@ CONFIG_VERSION_MANUFACTURER="Imagination Technologies"
 CONFIG_VERSION_MANUFACTURER_URL="www.imgtec.com"
 CONFIG_VERSION_PRODUCT="Creator Ci40(Marduk)"
 CONFIG_VERSION_HWREV="Ci40 with CA8210"
+CONFIG_DEVEL=y
```
We should not be adding hacks into this config file as the users may not use it if they build it manually, also upstream builds won't use it. It looks like this is done by adding 'fpu' as a feature in target/linux/pistachio/Makefile?

francois-berder committed:
pistachio: Use hard-float by default

Setting fpu in FEATURES enables the HAS_FPU option which
disables by default CONFIG_SOFT_FLOAT.

Signed-off-by: Francois Berder <francois.berder@imgtec.com>
